src/js/engagement_view/.yarnrc.yml

```diff
@@ -7,3 +7,6 @@ httpTimeout: 60000
 installStatePath: .yarn/state/install-state.gz
 
 nodeLinker: node-modules
+
+#Uncomment the following line for local development
+#yarnPath: .yarn/releases/yarn-3.1.1.cjs
```

Comment:
I'm about to put a PR up that will remove the need for this. You can delete it or leave it